### ISSUE TITLE
About LiCSBAS13_sb_inv.py

### ISSUE THREAD

**Issue by Freed-thinks** [yumorishita/LiCSBAS#322]
Dear Mr. Morishita,

Could you please provide guidance on the usage of the iCSBAS13_sb_inv.py script? There is a parameter named "--n_unw_r_thre" that I'd like clarification on.

`This parameter represents the threshold of "n_unw" (number of used unwrap data). Note that this value is a ratio to the number of images, specifically 1.5 times the number of images (i.e., 1.5 * n_im). A larger number (e.g., 2.5) speeds up processing but results in sparser output. The default values are 1 and 0.5 for C-band and L-band, respectively.`

Is "n_im" the number of unwrap images? For instance, if there are 10 unwrap images, the calculated value from the formula above would be 15. What does this number signify?

Alternatively, does "unwrap data" refer to the number of times unwrapping is performed per individual image?

**Comment by yumorishita** [comment]
No, `n_im` is the number of dates or epochs. For example, if you have 10 epochs (`n_im=10`) and and 3 ifg connections for each epoch, `n_unw=27`. If "--n_unw_r_thre=1.5`, `n_unw_thre=10*1.5=15`, meaning that pixels with `n_unw<15` (max 27) will be removed.

**Comment by ajisugiantoro123** [comment]
LiCSBAS12_loop_closure.py ver1.6.2 20230822 Y. Morishita
LiCSBAS12_loop_closure.py -d GEOCml1mask -l 3.8 --multi_prime

loop_thre : 3.8 rad

1st Loop closure check and make png for all possible 16 loops,
with 16 parallel processing...
    0/ 16th loop...

2nd Loop closure check without bad ifgs to define ref area...
with 16 parallel processing...
    0/ 16th loop...

3rd loop closure check taking into account ref phase...
with 16 parallel processing...
    0/ 16th loop...

4th loop to compute statistics...
with 16 parallel processing...
    0/ 16th loop...

Saving coh_avg, n_unw, and n_loop_err...

9/14 ifgs are discarded from further processing.
20230302_20230314
20230302_20230326
20230302_20230407
20230302_20230419
20230314_20230326
20230326_20230407
20230326_20230419
20230326_20230501
20230407_20230419

Connected network (year, n_image):
20230314-20230501 (0.13, 4)

If you want to change the bad ifgs to be discarded, re-run with different thresholds before next step.

Elapsed time: 00h 00m 06s

LiCSBAS12_loop_closure.py Successfully finished!!

Output directory: TS_GEOCml1mask


LiCSBAS13_sb_inv.py ver1.5.2 20210311 Y. Morishita
LiCSBAS13_sb_inv.py -d GEOCml1mask

In geographical coordinates
Read cum data on memory (fast but need memory).

Size of image (w,l)    : 2767, 2903
# of all images        : 6
# of images to be used : 4
# of all ifgs          : 14
# of ifgs to be used   : 5
# of removed ifgs      : 9
Threshold of used unw  : 4

Reference area (X/Y)   : 2221:2222/213:214
Allowed memory size    : 8000 MB
Number of patches      : 1
Inversion algorism     : LS
Gamma value            : 0.0001

Process 2903/2903th line (1/1th patch)...
  Reading 5 ifg's unw data...
  **7962539/8032601 points removed due to not enough ifg data...**

Hello Prof, I found that my pixels were almost all removed in step 13. How to reduce removed pixels in this step? And what do removed pixels mean? Does this mean there is no displacement? Thanks in advanced

**Comment by yumorishita** [comment]
This is because many unwrapping errors are included. Please see https://github.com/yumorishita/LiCSBAS/issues/244#issuecomment-1945899331.
